### PR TITLE
BZ1861066: added general SSH step and GCP-dspecific step to SSH module

### PR DESCRIPTION
**Pull request by skrthomas**
I added the requested info, but only added item 2 to GCP-related topics.

https://bugzilla.redhat.com/show_bug.cgi?id=1861066

Link to doc preview: https://deploy-preview-31723--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_gcp/installing-gcp-default.html#ssh-agent-using_installing-gcp-default

for 4.5+

### CODE REVIEW

**Comment by skrthomas** [comment]
Ready for QA @tsze-redhat

File: modules/ssh-agent-using.adoc [hunk 1]
```diff
@@ -148,7 +163,7 @@ following command:
 $ ssh-keygen -t ed25519 -N '' \
     -f <path>/<file_name> <1>
 ----
-<1> Specify the path and file name, such as `~/.ssh/id_rsa`, of the new SSH key.
+<1> Specify the path and file name, such as `~/.ssh/id_rsa`, of the new SSH key. If you have an existing key pair, make sure your public key is in the your ~/.ssh directory.
```
Comment (jboxman) [inline modules/ssh-agent-using.adoc:166]: Could be

> _Ensure_ your public key...

File: modules/ssh-agent-using.adoc [hunk 2]
```diff
@@ -180,6 +195,20 @@ Identity added: /home/<you>/<path>/<file_name> (<computer_name>)
...
 <1> Specify the path and file name for your SSH private key, such as `~/.ssh/id_rsa`
 
+ifdef::gcp[]
+. Set the `GOOGLE_APPLICATION_CREDENTIALS` environment variable to the full path to your service account private key file.
++
+[source,terminal]
+----
+$ export GOOGLE_APPLICATION_CREDENTIALS="<your_service_account_file>"
+----
+.. Verify that the credentials were applied.
```
Comment (jboxman) [inline modules/ssh-agent-using.adoc:205]: Does it make sense to have this as a second level step? `..` vs `.`

File: modules/ssh-agent-using.adoc (hunk 1 above)
Comment (jboxman) [inline modules/ssh-agent-using.adoc:166]: `~/.ssh`

**Review by jboxman — CHANGES_REQUESTED** [review]
Hi! I've left some comments for your review, thanks!

**Comment by tsze-redhat** [comment]
LGTM. Thanks.

File: modules/ssh-agent-using.adoc (hunk 2 above)
Comment (skrthomas) [reply modules/ssh-agent-using.adoc:205]: I was torn honestly. I'll just make it single level for consistency so it isn't an oddball.

**Review by jboxman — APPROVED** [review]

**Pull request merged**